config/games/secrethitler.ts

```diff
@@ -0,0 +1,39 @@
+import { ServerGame } from "../../types/types";
+import { postJson } from "../../utils/utils";
+
+const game: ServerGame = {
+    id: "secrethitler-duc",
+    name: "Secret Hitler",
+    author: "Duc Ngo Viet",
```

Comment:
We can supply information about the game on which this is based.

```suggestion
    author: "Duc Ngo Viet",
    basedOn: {
        game: "Secret Hitler",
        author: "Goat, Wolf, & Cabbage LLC",
        link: "https://www.secrethitler.com/",
        bggId: 188834,
    },
```

Reply:
thanks!